README.md

```diff
@@ -124,6 +124,14 @@ mkdir -p bin
 GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "-X main.Version=dev+git" -o bin/kd_linux_amd64
 ```
 
+You can also build `kd` in a docker container:
+
+```bash
+docker run --rm -v "$PWD":/go/src/UKHomeOffice/kd -w /go/src/UKHomeOffice/kd -ti golang:1.6 bash
```

Comment:
Use `${PWD}` instead of double quotes. Also it's `/go/src/github.com/UKHomeOffice/kd` and not `/go/src/UKHomeOffice/kd`.